Score correction made in Worcester St. @ WPI (index) quarter 3 drive 2
Final score WPI 31 - Worcester St. 20
Final score Worcester St. 19 - Union (N.Y.) 56
Final score Worcester St. 38 - Fitchburg St. 35
Final score Mass.-Dartmouth 52 - Worcester St. 7
Final score Worcester St. 13 - Bridgewater St. 49
Score correction made in Worcester St. @ Mass. Maritime (index) quarter 1 drive 6
Final score Mass. Maritime 24 - Worcester St. 28
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 97, in taster
    abbr = abbr_finder(quarters[0][drive_count],abbr)
IndexError: list index out of range
Final score Worcester St. 30 - Western Conn. St. 56
Score correction made in Worcester St. @ Westfield St. (index) quarter 2 drive 2
Score correction made in Worcester St. @ Westfield St. (index) quarter 2 drive 5
Score correction made in Worcester St. @ Westfield St. (index) quarter 3 drive 0
Score correction made in Worcester St. @ Westfield St. (index) quarter 3 drive 1
Score correction made in Worcester St. @ Westfield St. (index) quarter 3 drive 2
Score correction made in Worcester St. @ Westfield St. (index) quarter 3 drive 5
Final score Westfield St. 30 - Worcester St. 33
Final score Worcester St. 21 - Framingham St. 40
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 44, in taster
    name_dict[away_kicker] = kicker_names[0][0]
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 44, in taster
    name_dict[away_kicker] = kicker_names[0][0]
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 44, in taster
    name_dict[away_kicker] = kicker_names[0][0]
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 44, in taster
    name_dict[away_kicker] = kicker_names[0][0]
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 44, in taster
    name_dict[away_kicker] = kicker_names[0][0]
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 44, in taster
    name_dict[away_kicker] = kicker_names[0][0]
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 44, in taster
    name_dict[away_kicker] = kicker_names[0][0]
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 44, in taster
    name_dict[away_kicker] = kicker_names[0][0]
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 44, in taster
    name_dict[away_kicker] = kicker_names[0][0]
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 17, in taster
    name_dict[away_name] = team_names[0].string
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 44, in taster
    name_dict[away_kicker] = kicker_names[0][0]
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 44, in taster
    name_dict[away_kicker] = kicker_names[0][0]
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 44, in taster
    name_dict[away_kicker] = kicker_names[0][0]
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 44, in taster
    name_dict[away_kicker] = kicker_names[0][0]
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 44, in taster
    name_dict[away_kicker] = kicker_names[0][0]
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 44, in taster
    name_dict[away_kicker] = kicker_names[0][0]
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 44, in taster
    name_dict[away_kicker] = kicker_names[0][0]
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 44, in taster
    name_dict[away_kicker] = kicker_names[0][0]
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 44, in taster
    name_dict[away_kicker] = kicker_names[0][0]
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 44, in taster
    name_dict[away_kicker] = kicker_names[0][0]
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 44, in taster
    name_dict[away_kicker] = kicker_names[0][0]
IndexError: list index out of range
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 21, in pbp_parser
    name_dict[home_name] = drive_chart('a', href='#home-drives')[0].string
TypeError: 'NoneType' object is not callable
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 44, in taster
    name_dict[away_kicker] = kicker_names[0][0]
IndexError: list index out of range
